LayoutTests/imported/w3c/web-platform-tests/css/motion/offset-path-shape-ellipse-006.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>CSS Motion Path test: &lt;basic-shape&gt; ellipse() path with offset-distance</title>
+<!-- Chrome currently passes with 13 max pixel difference and 473 total different pixels -->
+<!-- You can increase the numbers to handle anti-aliasing -->
+<meta name="fuzzy" content="maxDifference=0-150; totalPixels=0-530">
+<link rel="author" title="Daniil Sakhapov" href="sakhapov@google.com">
+<link rel="match" href="offset-path-shape-ellipse-003-ref.html">
```

Comment:
If the ref is different than `offset-path-shape-ellipse-003-ref.html` please reflect it here. Otherwise if it's identical to `offset-path-shape-ellipse-003-ref.html`, please delete the `006-ref.html` file you added :)

Reply:
Fixed.